GG/src/ListBox.cpp

```diff
@@ -264,21 +264,25 @@ bool ListBox::Row::IsNormalized() const
 void ListBox::Row::Render()
 {}
 
+void ListBox::Row::GrowWidthsStretchesAlignmentsTo(std::size_t nn) {
+    while (m_col_widths.size() < nn) {
+        m_col_widths.emplace_back(X(5));
```

Comment:
why not use `vector::resize(nn, X(5))` and similar? the loop seems oddly verbose for this purpose. a check that `nn >= m_col_widths.size()` would more explicitly ensure no shrinkage if that is desired.